CHANGELOG.rst

```diff
@@ -600,8 +600,8 @@ Added/Improved Documentation
 Backwards Incompatible Changes
 ------------------------------
 
-- ``sunpy.instr`` has been depreacted and will be removed in sunpy 3.1 in favour of `sunkit_instruments`.
-  The code that is under ``sunpy.instr`` is imported via `sunkit_instruments` to ensure backwards comparability. (`#4526 <https://github.com/sunpy/sunpy/pull/4526>`__)
+- ``sunpy.instr`` has been depreacted and will be removed in sunpy 3.1 in favour of ``sunkit_instruments``.
```

Comment:
Is the better fix here make sure that `sunkit_instruments` is in intersphinx in `conf.py`?

Reply:
I thought it was, but I didn't check

Reply:
👍 I think updating `conf.py` is the better fix to do here.

Reply:
Its already there. So I don't know why it failed

Reply:
Hmm, it's failing on the normal doc bulids recently too. Maybe a change in `sunkit_instruments`?

Reply:
I think https://github.com/sunpy/sunkit-instruments/commit/0a76e4d75d5c28069ad5e2507c8a1bbb2198f27b#diff-1ab0170e21712558223705b73269528252b7f678af3241add91078655cbfc6f9L10 might have broke this - I'd leave the broken links as broken links in this PR, then we can backport a fix from `main`.

Reply:
Oh right, I can add that back